client/components/plans/plan-price/index.jsx

```diff
@@ -23,8 +23,9 @@ const PlanPrice = React.createClass( {
 				return this.translate( 'Free', { context: 'Zero cost product price' } );
 			}
 
-			const monthlyPrice = +( rawPrice / 12 ).toFixed( 2 );
-			formattedPrice = formattedPrice.replace( rawPrice, monthlyPrice );
+			const currencySymbol = formattedPrice.slice( 0, 1 );
+			const monthlyPrice = +( rawPrice / 12 ).toFixed( currencySymbol === '¥' ? 0 : 2 );
```

Comment:
Are we sure there are no other currencies/locales except JPY with a similar behavior? `currencySymbol === '¥' ? 0 : 2` seems a little too static though if JPY is the only one like this, it should be fine.